pkg/cmd/admin/node/evacuate.go

```diff
@@ -12,19 +12,32 @@ import (
 	kerrors "k8s.io/kubernetes/pkg/util/errors"
 )
 
+const (
+	flagGracePeriod = "grace-period"
+	flagDryRun      = "dry-run"
+	flagForce       = "force"
+
+	defaultGracePeriod = int64(30)
+	defaultDryRun      = false
+	defaultForce       = false
+)
+
 type EvacuateOptions struct {
 	Options *NodeOptions
 
 	// Optional params
-	DryRun bool
-	Force  bool
+	DryRun      bool
+	Force       bool
+	GracePeriod int64
 }
 
 func (e *EvacuateOptions) AddFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 
-	flags.BoolVar(&e.DryRun, "dry-run", false, "Show pods that will be migrated. Optional param for --evacuate")
-	flags.BoolVar(&e.Force, "force", false, "Delete pods not backed by replication controller. Optional param for --evacuate")
+	flags.BoolVar(&e.DryRun, flagDryRun, defaultDryRun, "Show pods that will be migrated. Optional param for --evacuate")
```

Comment:
Using the current value of `e.DryRun` makes it possible for different composers to choose different defaults.